go/chat/sender.go

```diff
@@ -281,10 +285,10 @@ func (s *Deliverer) deliverLoop() {
 		// Send messages
 		pops := 0
 		for _, obr := range obrs {
-			obr.Msg.ClientHeader.OutboxID = &obr.OutboxID
-			_, _, _, err := s.sender.Send(context.Background(), obr.ConvID, obr.Msg)
+			_, _, _, err := s.sender.Send(context.Background(), obr.ConvID, obr.Msg, 0)
```

Comment:
The "sender" being used here ignores that parameter (since it has already been filled in one layer up).